test/unit/url_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class UrlTest < Test::Unit::TestCase
+  context ".validate" do
+    should "return true for a valid url" do
+      assert Twingly::URL.validate("http://blog.twingly.com/"), "Should be valid"
+    end
+
+    should "return false for a valid url" do
```

Comment:
Fixed in https://github.com/twingly/twingly-url/commit/cce261155aeaba84994e54f543dc2dc7cbcfaeae